src/components/Settings.jsx

```diff
@@ -112,6 +121,54 @@ export default function Settings({ currentWallet }) {
             </rb.Dropdown.Item>
           </rb.Dropdown.Menu>
         </rb.Dropdown>
+
+        <br />
+
+        <rb.Button
```

Comment:
Last thing, can you remove the `<br />` wrap everything in a `<div>` please? The layout is off otherwis:



<img width="314" alt="Screenshot 2022-03-09 at 17 49 53" src="https://user-images.githubusercontent.com/10026790/157490017-d33d64bf-5cd5-4dd3-a787-e1382ed1ed87.png">

Reply:
Yes sorry I was not able to view the UI yet to see this because of my error.